bin/run-tests.sh

```diff
@@ -11,27 +11,23 @@
 # Example:
 # ./bin/run-tests.sh
 
-exit_code=0
+echo "current directory ${PWD}"
 
-# Iterate over all test directories
-for test_dir in tests/*; do
-    test_dir_name=$(basename "${test_dir}")
-    test_dir_path=$(realpath "${test_dir}")
+test_dir=tests/success
```

Comment:
This isn't looping over all the test directories anymore. Could you re-add that?